test/http_mock_adapter_test.dart

```diff
@@ -1,11 +1,84 @@
 import 'package:dio/dio.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:http_mock_adapter/http_mock_adapter.dart';
+import 'package:http_mock_adapter/src/interceptors/http_interceptor.dart';
 import 'package:http_mock_adapter/src/utils.dart';
 
 const path = 'https://example.com';
 
 void main() {
+  group("Interceptor", () {
+    MainInterceptor dioInterceptor;
+    setUp(() {
+      dioInterceptor = MainInterceptor();
+    });
+    int statusCode = 200;
+    final data = {'message': 'Test!'};
+
+    group("RequestRouted test for dioInterceptor", () {
```

Comment:
You are actually not actually fetching anything, you are simply declaring routes, which do pass the tests, but they should also be checked and compared to actual results.

Similar, updated code is written in #22 which improves on `onRoute()` and its wrapper method testings. They could be of use.